omod/src/main/java/org/openmrs/module/reportingrest/web/wrapper/ScheduledReport.java

```diff
@@ -0,0 +1,46 @@
+/**
```

Comment:
This class should not be needed.  A scheduled report is just a report request that has a non-null / non-blank schedule property.

Reply:
I've removed this ScheduledReport class and replaced by SimpleObject according to your one of above comments.